diff --git a/src/commons.h b/src/commons.h
index 6bdf88be..8751de38 100644
--- a/src/commons.h
+++ b/src/commons.h
@@ -44,7 +44,7 @@
 #define GO_UNUSED __attribute__((unused))
 #define GO_VERSION 		"1.3"
 #define GO_WEBSITE 		"http://goaccess.io/"
-struct tm *now_tm;
+extern struct tm *now_tm;
 
 /* common char array buffer size */
 #define INIT_BUF_SIZE 1024
diff --git a/src/csv.c b/src/csv.c
index fefaa0e0..9ef74c85 100644
--- a/src/csv.c
+++ b/src/csv.c
@@ -55,6 +55,8 @@
 #include "ui.h"
 #include "util.h"
 
+struct tm *now_tm;
+
 /* Panel output */
 typedef struct GPanel_
 {
